.github/workflows/buildAndTest.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build And Test
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - uses: actions/cache@v1
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+
+    - name: Unit Tests
```

Comment:
Did you remove the build step now?

Reply:
Travis folded it into `mvn test`; we likely don't need a standalone build step.

Reply:
Yes, `test` depends on `compile`